mechanical/rocker_bogie/README.md

```diff
@@ -111,9 +111,9 @@ Attach another 3.75 inch channel **S3** to the bottom hole of the pattern F brac
 
 ### 4.5 Opposite edge of Rocker-Bogie Joint
 
-Attach another Pattern F bracket **S7** to the opposite side of the channels using washers **W1** and **W3**, screws **B4**, and hex nuts **B8** as shown in Figure 7.
+Attach another Pattern F bracket **S7** to the opposite side of the channels using washers **W1** and **W3**, screws **B2**, and hex nuts **B8** as shown in Figure 7.
```

Comment:
If we do make this change, we have to make sure that the tables containing the quantities are updated as well (if they are incorrect).

Reply:
Good call on the table, I updated the numbers.